release/fabfile.py

```diff
@@ -563,8 +570,9 @@ def tag(name):
             name = get_tarball_name(key)
             with tag('tr'):
                 with tag('td'):
-                    with tag('b'):
-                        table.append(name)
+                    with a_href('https://github.com/sympy/sympy/releases/download/sympy-'+'%s' %version+'/'+'%s' %name):
```

Comment:
Also here you can just use one string. `'https://github.com/sympy/sympy/releases/download/sympy-%s/%s' % (version, name)`.